api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -271,10 +272,20 @@ public async Task<IActionResult> UpdateAsync(ContentModel model, bool index = fa
     [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Content" })]
-    public IActionResult UpdateStatusAsync(ContentModel model)
+    public async Task<IActionResult> UpdateStatusAsync(ContentModel model)
```

Comment:
@kylgarmor I fixed several regression issues introduced in a prior PR.
1) Async function naming
2) SignalR message when status has been updated
3) Update status in database